agent/iptables_helper.py

```diff
@@ -146,7 +145,8 @@ def block_networks(network_list: List[Tuple[str, bool]]):
     add_rules(TABLE, OUTPUT_CHAIN, rules)
 
 
-def block(blocklist, debug):
+def block(blocklist):
+    logger = logging.getLogger(__name__)
```

Comment:
Such objects should be created only once. And this code should be placed on the top level of a module.

Reply:
fixed